### ISSUE TITLE
Usage for command should include parent commands

### ISSUE THREAD

**Issue by mojavelinux** [tj/commander.js#739]
The usage for an action-based (sub)command is misleading. It seems to suggest that the command can be used by itself, without the parent.

For example, if we were making the git cli with commander.js, the usage for `add` would look like:

```
Usage: add [options] <pathspec>
```

However, it should show the parent command (the context) as well:

```
Usage: git add [options] <pathspec>
```

I guess if it's an executable it's fine, so this just applies to subcommands that are not executables.

**Comment by privateOmega** [comment]
Please merge this.

**Comment by shadowspawn** [comment]
This issue will be resolved when v3.0.0 is released. Available now as a prerelease. See #1001

**Comment by shadowspawn** [comment]
Shipped in v3 (thanks @mojavelinux): https://github.com/tj/commander.js/releases/tag/v3.0.0
